doc/en/deprecations.rst

```diff
@@ -20,6 +20,20 @@ Below is a complete list of all pytest features which are considered deprecated.
 :ref:`standard warning filters <warnings>`.
 
 
+``--no-print-logs`` command-line option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. deprecated:: 5.4
+
+
+Option ``--no-print-logs`` is deprecated and meant to be removed in a future release. If you use ``--no-print-logs``, please try out ``--show-capture`` and
+provide feedback.
+
+``--show-capture`` command-line option was added in ``pytest 3.5.0` and allows to specify how to
```

Comment:
Hi there,
all the options aren't documented in the [latest docs](https://docs.pytest.org/en/latest/capture.html#setting-capturing-methods-or-disabling-capturing) at the time of writing.
However the options mentioned here do seem to work regardless.

i.e. 
`pytest --show-capture=stdout` does work but isn't mentioned in the docs.

Reply:
Mixed up `show-capture` and `capture`. Deleting comments :)